bundles/org.openhab.binding.evcc/src/main/java/org/openhab/binding/evcc/internal/EvccBindingConstants.java

```diff
@@ -21,19 +21,27 @@
  * used across the whole binding.
  *
  * @author Florian Hotze - Initial contribution
+ * @author Luca Arnecke - Update to evcc version 0.123.1
```

Comment:
I would only mention that for important and large code changes (i.e. the changes to the handler), at least this seems to be the style at openhab-addons.

```suggestion
```

Reply:
this is not my call. Luca took the initial action on this PR. I don't want to drop him from the history if he was adding himself.

Reply:
Okay, in that case, just leave it there to get this finished.